applications/app/controllers/AllIndexController.scala

```diff
@@ -55,9 +55,9 @@ object AllIndexController extends Controller with ExecutionContexts with ItemRes
 
     loadLatest(path, reqDate).map { _.map { index =>
 
-      val contentOnRequestedDate = index.trails.filter(_.webPublicationDate.sameDay(reqDate))
+      val contentOnRequestedDate = index.contents.filter(_.item.trail.webPublicationDate.sameDay(reqDate))
 
-      val olderDate = index.trails.find(!_.webPublicationDate.sameDay(reqDate)).map(_.webPublicationDate.toDateTime)
+      val olderDate = index.trails.find(!_.trail.webPublicationDate.sameDay(reqDate)).map(_.trail.webPublicationDate.toDateTime)
 
```

Comment:
Good question. A `trail` used to be a trait, defining all the fields needed to render a html trail link. Now, it is simply part of `content`, although still represents the same data fields.